python/pyxet/pyxet/cli.py

```diff
@@ -179,7 +179,8 @@ def ls(path: Annotated[str, typer.Argument(help="Source file or folder to list")
                 print(tabulate(listing, headers="keys"))
             return listing
         except Exception as e:
-            raise
+            print(f"{e}")
+            return
```

Comment:
The problem with just return here is that the program as a whole will report success through the exit code.  We should probably use sys.exit(1)

Reply:
Gotcha!